news/_posts/2020-05-11-cms-security-summit.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title:  "CMS Security Summit 2020 with Google"
+subtitle: "PrestaShop and CMS work together with Google for a safer web"
+date:   2020-06-04 17:00:00
+authors: [ PierreRambaud ]
+image: /assets/images/theme/meta-logo-build.png
+icon: icon-padlock
+tags: [security]
+---
+
+[Once again](https://build.prestashop.com/news/we-were-at-the-cms-security-summit-with-google/), open source CMSs Security leads shared knowledge together at an event organized by Google in 2020 in Munich.
+
+
+# Lightning talks & Security technologies
+
+After spending the first evening with stakeholders and meeting the newcomers over a few beers, we spent two days at Google's Munich office and [Information Security Hub (ISH)](https://www.ish-muc.com/) to talk about what we have achieved since last year and about what's coming.
```

Comment:
```suggestion
After spending the first evening with stakeholders and meeting the newcomers over a few beers, the attendees and I spent two days at Google's Munich office and [Information Security Hub (ISH)](https://www.ish-muc.com/) to talk about what we have achieved since last year and about what's coming.
```

just to clarify who is we.